app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -318,7 +318,22 @@ class MigrationsProvider(
 
     val MIGRATION_23_TO_24: Migration = object : Migration(23, 24) {
         override fun migrate(database: SupportSQLiteDatabase) {
-            database.execSQL("ALTER TABLE `tabs` ADD COLUMN `sourceTabId` TEXT")
+            //https://stackoverflow.com/a/57797179/980345
```

Comment:
@CDRussell When testing migrations I detected an issue trying to `ALTER` a table to introduce a `FOREIGN` key. It seems it's not supported and the workaround is to migrate -> drop -> create a new table.

I have my worries (based on nothing in concrete, just a feeling), don't know if this can be an issue for users with many open tabs...